tests/postman.collection.Transmission.json

```diff
@@ -1036,6 +1036,32 @@
                   }
                 },
                 "response": []
+              },
+              {
+                "name": "Get List of devices",
```

Comment:
I'd change the name to be a bit clearer on what exactly you're retrieving.

Reply:
What about `List Organization devices`?